### PR TITLE
Deprecations of k8ssandra 1.x.

### PR DESCRIPTION
**Pull request by Miles-Garnsey**
**What this PR does**:

What it says on the tin.

**Which issue(s) this PR fixes**:
#1537 

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: CHANGELOG-1.6.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ and date `## vX.Y.Z - YYYY-MM-DD` and create a new placeholder section for  `unr
 
 * [CHANGE] Upgrade Stargate to v1.0.63
 * [CHANGE] Upgrade cass-operator to v1.10.4
+* [CHANGE] Deprecate all K8ssandra v1.x charts, refer users to k8ssandra-operator.
+* [CHANGE] Upgrade Stargate to v1.0.61
+* [CHANGE] Upgrade cass-operator to v1.12.0
```
Comment (burmanm) [inline CHANGELOG-1.6.md:25]: k8ssandra 1.x does not use 1.12.0 (it should use 1.10.5), this shouldn't be part of the changelog.

File: charts/k8ssandra/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 apiVersion: v2
 name: k8ssandra
 description: |
+  DEPRECATED please see https://github.com/k8ssandra/k8ssandra-operator
   Provisions and configures an instance of the entire K8ssandra stack. This includes Apache Cassandra, Stargate, Reaper, Medusa, Prometheus, and Grafana.
```
Comment (jsanda) [inline charts/k8ssandra/Chart.yaml:5]: Might want to reword the description slightly since k8ssandra-operator does not provision Prometheus or Grafana.

Comment (Miles-Garnsey) [reply charts/k8ssandra/Chart.yaml:5]: That's for the k8ssandra-operator chart right? Not the k8ssandra chart?

File: CHANGELOG-1.6.md (hunk 1 above)
Comment (Miles-Garnsey) [reply CHANGELOG-1.6.md:25]: That was part of the rebase I did. Maybe I shouldn't be branching off master and should branch off a release branch? Some of our projects still use tags on main and some use release branches, what's our strategy on this one?

**Comment by Miles-Garnsey** [comment]
This was failing earlier because that `github.com/gruntwork-io/terratest/modules/helm` library was somehow rendering both stdout and stderr into the same output stream. As a result, the deprecation warning was being picked up as if it was part of the yaml.

Clearly this is totally whack, I've tried to shim it out by filtering out that line from the rendered output, it isn't pretty but I don't see any alternative other than eliminating that library from the tests.

File: charts/k8ssandra/Chart.yaml (hunk 2 above)
Comment (jsanda) [reply charts/k8ssandra/Chart.yaml:5]: Yup, I got my wires crossed. Sorry about that.

File: CHANGELOG-1.6.md (hunk 1 above)
Comment (jsanda) [reply CHANGELOG-1.6.md:25]: >That was part of the rebase I did

I just looked at `charts/k8ssandra/Chart.yaml` in the PR and it still depends on version 0.35.3 of the cass-operator chart, which installs cass-operator 1.10.4. Due to API changes in Kubernetes in 1.22 I believe I think we have avoided upgrading the k8ssandra chart to 1.11 or later of cass-operator.

Comment (Miles-Garnsey) [reply CHANGELOG-1.6.md:25]: OK, tbh I don't know where these changes came from, I'm just going to remove them as you ask. We have another convo going offline about the fact that we're still supporting cass-operator 1.10.x.

**Review by jsanda — APPROVED** [review]

**Pull request merged**